rocky/katalogus/client.py

```diff
@@ -195,10 +202,13 @@ def parse_normalizer(normalizer: Dict) -> Normalizer:
     name = normalizer["id"].replace("_", " ").replace("kat ", "").title()
 
     consumes = set(normalizer["consumes"])
+    consumes.add(f"normalizer/{name.lower()}")
```

Comment:
This should be

```suggestion
    consumes.add(f"normalizer/{normalizer['id'}")
```

to match https://github.com/minvws/nl-kat-coordination/blob/main/boefjes/boefjes/plugins/models.py#L69

But given that the katalogus adds it when reading the normalizer definition, shouldn't this already be included?

Reply:
There are at least some issues where the mime-type for a normaliser (as listed on this page) looses its underscores where it shouldn't.